### PR TITLE
Improvement/#117 disattenuated correlations

### PR DESCRIPTION
**Pull request by aloukina**
* Added disattenuated correlations to consistency notebook in RSMTool
* Added disattenuated correlations to consistency notebook in RSMCompare
* Added new outputs to all tests and updated list of expected files
* Added new test for RSMCompare to include both h2 and subgroups

* Fixed paths in `test_utils.py` to make it possible to run the tests from a different working directory

* Copied from `master` two utility scripts for updating the tests and made edits necessary given new test structure

* Updated documentation to include disattenuated correlations
* Updated documentation to make reference to .csv files generic since we now allow various types of files

### CODE REVIEW

File: rsmtool/analyzer.py [hunk 1]
```diff
@@ -701,6 +701,55 @@ def metrics_helper(human_scores,
...
+                                           human_human_corr):
+        """
+        Compute the disattenuated correlations between human and machine scores. 
+        These are computed as the Pearson's correlation between the human score
+        and the machine score divided by the square root of correlation between
+        two human raters. 
+
+        Parameters
+        ----------
+        human_machine_corr : pandas DataFrame
```
Comment (jbiggsets) [inline rsmtool/analyzer.py]: Should this be pandas Series?

Comment (aloukina) [reply rsmtool/analyzer.py]: Yes!

File: rsmtool/analyzer.py [hunk 2]
```diff
@@ -1358,10 +1426,12 @@ def run_prediction_analyses(self,
             - eval_short
             - consistency
             - degradation
+            - disattenudated_correlations
             - confMatrix
             - score_dist
             - eval_by_*
             - consistency_by_*
+            - disattenduated_correlations_by*
```
Comment (jbiggsets) [inline rsmtool/analyzer.py]: _by_*

File: rsmtool/comparer.py [hunk 3]
```diff
@@ -315,6 +315,28 @@ def load_rsmtool_output(self, filedir, figdir, experiment_id, prefix, groups_eva
...
+
+        # load if disattenuated correlations is present
+        if exists(dis_corr_file):
+            df_dis_corr = DataReader.read_from_file(dis_corr_file, index_col=0)
+            # we only use the row for raw_trim or scale_trim score
+            files['df_disattenuated_correlations'] = df_dis_corr.loc[['{}_trim'.format(prefix)]]
+
+        # read in disattenuated correlations by group
+        for group in groups_eval:
+          group_dis_corr_file = join(filedir,
```
Comment (jbiggsets) [inline rsmtool/comparer.py]: Is this four spaces or two?

**Pull request merged**